ikarus/finiteelements/mechanics/enhancedassumedstrains.hh

```diff
@@ -57,7 +57,17 @@ public:
   using GridView  = typename Traits::GridView;
   using Pre       = EnhancedAssumedStrainsPre;
 
-  using SupportedResultTypes = std::tuple<decltype(makeRT<ResultTypes::linearStress>())>;
+  using SupportedResultTypes = typename FE::SupportedResultTypes;
+
+  /**
+   * \brief Returns whether an element can provide a requested result. Can be used in constant expressions
+   * \tparam RT The type representing the requested result.
+   * \return boolean indicating if a requested result can be provided
+   */
+  template <template <typename, int, int> class RT>
+  static consteval bool canProvideResultType() {
```

Comment:
Is this function necessary? May be `isSupportedResultType` can directly be used with `calculateAtImpl`?

Reply:
I think so too. This function is not really necessary